CTCommons/MSTranscription/MSTranscriptionService.cs

```diff
@@ -35,6 +35,7 @@ public MSTranscriptionService(ILogger<MSTranscriptionService> logger, SlackLogge
             AppSettings _appSettings = Globals.appSettings;
 
             SpeechTranslationConfig _speechConfig = SpeechTranslationConfig.FromSubscription(key.ApiKey, key.Region);
+            //_speechConfig.EndpointId = "1b8a39b2-8718-413c-9a31-a77078fea5b8";
```

Comment:
Remove this, its generally a good idea to review each line of change before committing. :)